WordPress/Classes/ViewRelated/Menus/MenuDetailsViewController.m

```diff
@@ -43,15 +44,37 @@ - (void)viewDidLoad
     [self setupTextFieldDesignViews];
     [self setupDoneButton];
     [self setupTrashButton];
+    [self setupObservers];
 
     [self updateTextFieldDesignIconPositioning];
 }
 
+-(void)dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+- (void)contentSizeCategoryDidChange
+{
+    [self updateTextFieldFont];
+    [self updateTextFieldDesignIconPositioning];
+}
+
+- (void)setupObservers
```

Comment:
During my latest round of fights with dynamic type, I found that we can achieve the same result (being notified of changes in content size) by means of overriding `traitCollectionDidChange`. In my opinion it makes reacting to changes in text size slightly cleaner, and it does not require observing any notification.